urnai/models/model_base.py

```diff
@@ -0,0 +1,28 @@
+from abc import ABC, abstractmethod
+
+from urnai.utils.returns import ActionIndex
+
+
+class ModelBase(ABC):
+
+	learning_data = {}
```

Comment:
This is one of Python's little tricks.

![ModelBase-test](https://github.com/UFRN-URNAI/urnai-tools/assets/10817238/26f7a3f1-4285-47e0-8222-7ef3a9ae9ce3)

Suggestion:

![ModelBase-suggestion](https://github.com/UFRN-URNAI/urnai-tools/assets/10817238/35c845b3-e765-4bd2-aa5a-4457884538a8)